src/bindings/js/node/include/tensor.hpp

```diff
@@ -64,6 +64,12 @@ class TensorWrap : public Napi::ObjectWrap<TensorWrap> {
      * @return Napi::Boolean
     */
     Napi::Value is_continuous(const Napi::CallbackInfo& info);
+    /**
+     *@brief Sets the shape of the tensor
+     * @param info The contents of the parameters passed by the JS function.
+     * @return `null` if successful, otherwise returns an error.
+     */
+    Napi::Value set_shape(const Napi::CallbackInfo& info);
```

Comment:
`set_shape` should be void method as [here](https://github.com/openvinotoolkit/openvino/blob/e11eb21cb147190b4defdaac34c07becdbc4c4c1/src/core/include/openvino/runtime/tensor.hpp#L156)